app/src/Services/modules/projects/index.ts

```diff
@@ -24,12 +23,13 @@ export interface Project {
   client: string
   role: string
   description: string
-  video: ImageURISource
+  video_webPage: string
```

Comment:
I think this needs to be `video_webpage` rather than `video_webPage` as that's how it's named in the data coming from the API